HowItWorks.md

```diff
@@ -3,11 +3,9 @@ How it Works
 
 ![NVTabular Workflow](./images/nvt_workflow.png)
 
-NVTabular wraps the RAPIDS cuDF library which provides the bulk of the functionality, accelerating dataframe operations on the GPU.  We found in our internal usage of cuDF on massive datasets like [Criteo](https://labs.criteo.com/2014/02/kaggle-display-advertising-challenge-dataset/) or [RecSys 2020](https://recsys-twitter.com/) that it wasn’t straightforward to use once the dataset had scaled past GPU memory.  The same design pattern kept emerging for us and we decided to package it up as NVTabular in order to make tabular data workflows simpler.
+With the transition to v0.2 the NVTabular engine uses the [RAPIDS](http://www.rapids.ai) [Dask-cuDF library](https://github.com/rapidsai/dask-cuda) which provides the bulk of the functionality, accelerating dataframe operations on the GPU, and scaling across multiple GPUs.  NVTabular provides functionality commonly found in deep learning recommendation workflows, allowing you to focus on what you want to do with your data, not how you need to do it.  We also provide a template for our core compute mechanism, Operations, or ‘ops’ allowing you to build your own custom ops from cuDF and other libraries.
```

Comment:
The Dask-CuDF link is actually pointing to the Dask-CUDA library.  Since Dask-CuDF is actually a part of the CuDF repository, there is not a great landing page at the moment.  For now, it may be best to point to: https://github.com/rapidsai/cudf/tree/main/python/dask_cudf

I'll submit a small PR with the change - but wanted to make a quick note here in case I got pulled away.